app/views/grant_submissions/submissions/index.xlsx.axlsx

```diff
@@ -0,0 +1,73 @@
+
+base_headers_row_2 = ["Applicant",
```

Comment:
Why `row_2`?

Reply:
This is a hold over from when I added sections but then I removed the sections after talking to Pearl. Thanks for catching this.